tests/pass-dep/tokio_mvp.rs

```diff
@@ -0,0 +1,9 @@
+//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance
+//@only-target-linux: the errors differ too much between platforms
+
+#[cfg(not(target_arch = "x86_64"))]
```

Comment:
This is effectively not testing anything, is it? Why not use `only-target-x86_64` if we don't want to test other targets?